osis-core/src/test/java/com/scality/osis/security/crypto/AES256GCMTest.java

```diff
@@ -0,0 +1,62 @@
+package com.scality.osis.security.crypto;
+
+import com.scality.osis.security.crypto.model.SecretKeyRepoData;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.crypto.AEADBadTagException;
+import java.security.InvalidKeyException;
+import java.security.SecureRandom;
+
+import static com.scality.osis.utils.ScalityTestUtils.TEST_ACCESS_KEY;
+import static com.scality.osis.utils.ScalityTestUtils.TEST_SECRET_KEY;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class AES256GCMTest {
+
+    private AES256GCM aes256GCMUnderTest;
+
+    @BeforeEach
+    public void setUp() {
+        aes256GCMUnderTest = new AES256GCM();
+    }
+
+    @Test
+    public void testEncryptDecrypt() throws Exception {
+        //create new random key
+        final String key = RandomStringUtils.random(32, 0, 0, true, true, null, new SecureRandom());
```

Comment:
It's more of a personal preference matter, but I like to have hardcoded values in tests, to decrease the amount of moving parts.

Reply:
Fixed in 6a2f63423cce10658fa40363429704570b7528fb. Great catch